src/FormControl/FormControl.tsx

```diff
@@ -137,30 +93,12 @@ const FormControl = React.forwardRef<HTMLDivElement, FormControlProps>(
           id,
           required,
           validationMessageId,
+          validationStatus,
         }}
       >
         {isChoiceInput || layout === 'horizontal' ? (
           <Box ref={ref} display="flex" alignItems={slots.leadingVisual ? 'center' : undefined} sx={sx}>
-            <Box sx={{'> input': {marginLeft: 0, marginRight: 0}}}>
-              {React.isValidElement(InputComponent) &&
-                React.cloneElement(
-                  InputComponent as React.ReactElement<{
-                    id: string
-                    disabled: boolean
-                    ['aria-describedby']: string
-                  }>,
-                  {
-                    id,
-                    disabled,
-                    ['aria-describedby']: captionId as string,
-                  },
-                )}
-              {childrenWithoutSlots.filter(
-                child =>
-                  React.isValidElement(child) &&
-                  ![Checkbox, Radio].some(inputComponent => child.type === inputComponent),
-              )}
-            </Box>
```

Comment:
This change also fixes an apparently previously overlooked bug where `horizontal`-layout `FormControl` would not forward the `validationStatus` or `required` props

Reply:
This was not a bug, this was intentional. You can't require an individual `Radio` or `Checkbox`, and they can't have their own validation status.

Instead, you handle those on the `CheckboxGroup` or `RadioGroup` level.

See the following documentation for more info:
- https://primer.style/design/ui-patterns/forms/overview#validation-message
- https://primer.style/design/ui-patterns/forms/overview#validation-message:~:text=An%20individual%20checkbox%20or%20radio%20button%20cannot%20be%20marked%20as%20required.